py3-gcovr.yaml

```diff
@@ -0,0 +1,45 @@
+# Generated from https://pypi.org/project/gcovr/
+package:
+  name: py3-gcovr
+  version: "6.0"
+  epoch: 0
+  description: Generate C/C++ code coverage reports with gcov
+  copyright:
+    - license: BSD-3-Clause # according to https://github.com/gcovr/gcovr/tree/master#license
+  dependencies:
+    runtime:
+      - py3-jinja2
+      - py3-lxml
+      - py3-pygments
+      - python-3.11
```

Comment:
Does it need 3.11 specifically? If so, can you add a comment so someone (me) doesn't try to bump this in the future?

Reply:
No, `python3` is alright. That was generated by `melange convert`, I'll update.

Reply:
Updated to use `python3` instead.